cmake/catkin_package.cmake

```diff
@@ -75,6 +77,7 @@
 #     LIBRARIES projlib1 projlib2
 #     CATKIN_DEPENDS roscpp
 #     DEPENDS Eigen
+#     DEFINES MYDEFINE=1
```

Comment:
Should this be named `COMPILE_OPTIONS` to match the underlying variable as well as used CMake function? Then defines would need to be passed as `-Dkey=value` but it allows to pass anything else too.